File: voice-telephony/call-log/delete-user-call-log/code-samples/deleteUserCallLog.php
<?php
// https://developers.ringcentral.com/my-account.html#/applications
// Find your credentials at the above url, set them as environment variables, or enter them below

// PATH PARAMETERS
$accountId = '<ENTER VALUE>';
$extensionId = '<ENTER VALUE>';

// OPTIONAL QUERY PARAMETERS
$queryParams = array(
    //'dateTo' => '<ENTER VALUE>',
    //'phoneNumber' => '<ENTER VALUE>',
    //'extensionNumber' => '<ENTER VALUE>',
    //'type' => array( 'Voice', 'Fax' ),
    //'direction' => array( 'Inbound', 'Outbound' ),
    //'dateFrom' => '<ENTER VALUE>'
);

require('vendor/autoload.php');
$rcsdk = new RingCentral\SDK\SDK(getenv('clientId'), getenv('clientSecret'), getenv('serverURL'));
$platform = $rcsdk->platform();
$platform->login(getenv('username'), getenv('extension'), getenv('password'));
$r = $platform->delete("/restapi/v1.0/account/{$accountId}/extension/{$extensionId}/call-log", $queryParams);
// PROCESS RESPONSE
?>

File: voice-telephony/call-log/delete-user-call-log/code-samples/deleteUserCallLog.cs
// https://developers.ringcentral.com/my-account.html#/applications
// Find your credentials at the above url, set them as environment variables, or enter them below

// PATH PARAMETERS
string accountId = "<ENTER VALUE>";
string extensionId = "<ENTER VALUE>";

// OPTIONAL QUERY PARAMETERS
DeleteUserCallLogParameters deleteUserCallLogParameters = new DeleteUserCallLogParameters {
    //dateTo = "<ENTER VALUE>",
    //phoneNumber = "<ENTER VALUE>",
    //extensionNumber = "<ENTER VALUE>",
    //type = new[] { "Voice", "Fax" },
    //direction = new[] { "Inbound", "Outbound" },
    //dateFrom = "<ENTER VALUE>"
};

RestClient rc = new RestClient(
    Environment.GetEnvironmentVariable("clientId"),
    Environment.GetEnvironmentVariable("clientSecret"),
    false
);
await rc.Authorize(
    Environment.GetEnvironmentVariable("username"),
    Environment.GetEnvironmentVariable("extension"),
    Environment.GetEnvironmentVariable("password")
);
var r = await rc.Restapi().Account(accountId).Extension(extensionId).CallLog().Delete(deleteUserCallLogParameters);
// PROCESS RESPONSE
